### PR TITLE
Improves readme

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
I was going for delightful, but lovely will do too :D

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+For example (some labels ommited for clarity):
+```
+dns_latency_seconds_bucket{instance="192.168.120.239:8000", host="kubernetes.default.svc.cluster.local", le="0.008"} | 7
+```
+Here, we expose the latency for the specific instance to resolve the dns host.
+
+```
+network_latency_seconds_bucket{instance="192.168.120.239:8000", host="192.168.120.239:8000", le="0.004"} | 28
+```
+Here, we expose the latency for the specific instance to resolve another instance (specifically, the net-exporter pod, labeled as host).
```
Comment (kopiczko) [inline README.md]: For my interest: what happens for instance running the net-exporter?

Comment (JosephSalisbury) [reply README.md]: We run net-exporter as a daemonset, so essentially all instances run net-exporter. All instances ping all other instances.

Comment (kopiczko) [reply README.md]: Cool.

Comment (JosephSalisbury) [reply README.md]: This might not be super useful. Possibly a good idea to exclude an instance pinging itself.

**Pull request merged**